packages/muelu/src/Utils/MueLu_Utilities_decl.hpp

```diff
@@ -892,10 +892,34 @@ namespace MueLu {
 
 
 
-  /*! Removes the following non-serializable data (A,P,R,Nullspace,Coordinates) from level-specific sublists from inList
-    and moves it to nonSerialList.  Everything else is copied to serialList.  This function returns the level number of the highest level
-    for which non-serializable data was provided.
-    */
+  /*!
+  \bfief Extract non-serializable data from level-specific sublist and move to separate parameter list
```

Comment:
bfief should be replaced by brief

level-specific sublist 
should be replaced by 
level-specific sublists

and potentially
 move to separate parameter list 
could be replaced by
 move it to a separate parameter list